CONTRIBUTING.md

```diff
@@ -9,18 +9,15 @@ All the tools require an Linux Foundation (LF) ID and Github ID.
 
 If you do not have an LF ID, can [apply one](https://identity.linuxfoundation.org) for free.
 
-## Jira board usage
+## GitHub Issues
 
-We are using [Jira](https://jira.hyperledger.org/secure/RapidBoard.jspa?rapidView=85) to track the project progress, and welcome to report bug issues or create to-do tasks there. Each item should try keeping simple and focused, hence easy to fix and review.
+We are using [GitHub Issues](https://github.com/hyperledger/fabric-sdk-py/issues) for issue management and you are welcome to report bug issues there.
```

Comment:
@RohanShrothrium Could you just add the github but keep the jira for a while? Thanks!

Reply:
Sure will do the same!